### PR TITLE
New date proposal

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: distsys/20200218.md [hunk 1]
```diff
@@ -52,6 +52,6 @@ Here's a list of topics we touched on in no particular order.
     - with a person responsible to keep the conversation ordered and fluent
 - next steps:
     - meetup per month
-    - next will be remote on Wednesday 2020-03-18 at 13:30 at [zoom#605885166](https://zoom.us/j/605885166)
+    - next will be remote on Wednesday 2020-03-25 at 18:00 at [zoom#605885166](https://zoom.us/j/605885166)
```
Comment (dgomezferro) [inline distsys/20200218.md]: 19:00 would be ideal for me, but I might be able to join nevertheless.

Comment (gonzaloserrano) [reply distsys/20200218.md]: 18:30? :-)

Comment (gonzaloserrano) [reply distsys/20200218.md]: Hey everyone, I'm having a hard time concentrating on this, so I propose to skip it or do it without me. I'm sorry.

Comment (ignasi35) [reply distsys/20200218.md]: Take care @gonzaloserrano!

We'd need a new host/moderator. I think we should hold on any date/time until someone steps up as moderator.

Comment (ruben-perez) [reply distsys/20200218.md]: @gonzaloserrano 🤗🤗🤗🤗

Comment (ignasi35) [reply distsys/20200218.md]: Some family developments impend me from participating or even providing the Zoom room until further notice. Apologies.

Comment (ignasi35) [reply distsys/20200218.md]:
```suggestion
    - next will be remote on Wednesday 2020-03-25 at 18:00 at <Skype/hangouts/Jitsi/zoom required>.
```

File: distsys/20200218.md [hunk 2]
```diff
@@ -52,6 +52,6 @@ Here's a list of topics we touched on in no particular order.
     - with a person responsible to keep the conversation ordered and fluent
 - next steps:
     - meetup per month
-    - next will be remote on Wednesday 2020-03-18 at 13:30 at [zoom#605885166](https://zoom.us/j/605885166)
+    - next will be remote on Wednesday 2020-03-25 at 18:00 at <Skype/hangouts/Jitsi/zoom required>.
```
Comment (ignasi35) [inline distsys/20200218.md]:
```suggestion
    - next will be remote <sometime when conditions allow> at <Skype/hangouts/Jitsi/zoom required>.
```

**Pull request merged**